scripts/travis-ci/create-release-tarball.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh -e
+
+# This script will create a release package suitable for use on production
+# instances. This package excludes all tests and development tools.
+
+# Define paths.
+PROJECT_ROOT=$(pwd)
+RELEASE_PATH=${PROJECT_ROOT}/oe_theme
+
+# Get the release number.
+VERSION=$(git describe --tags)
+if [ $? -ne 0 ]; then
+  echo "No release tag associated with the current HEAD."
+  exit 1
+fi
+
+cd ${PROJECT_ROOT}
+
+# Create release folder.
+mkdir ${RELEASE_PATH}
+
+# Undo vandalism to composer.json.
+git checkout composer.json
+rm composer.lock
+
+# Install production dependencies.
+# Todo: Not needed if we exclude the vendor folder.
+sudo chmod -R a+w build/
+rm -rf build/
+rm -rf vendor/
+docker-compose exec -u web web composer install --no-dev
+
+# Copy production files.
+# Todo: exclude the vendor folder?
+cp -r \
+  config/ \
+  css/ \
+  js/ \
+  modules/ \
+  templates/ \
+  vendor/ \
```

Comment:
If we decide to not keep the locally installed dependencies then this line should also be removed.